src/components/Accordion.js

```diff
@@ -47,8 +47,8 @@ const Accordion = React.createClass({
         Svg("arrow", { className: opened[i] ? "expanded" : "" }),
         item.header,
         item.buttons ?
-        dom.button({ className: "header-buttons" },
-          item.buttons.map((button, id) => span({ key: id }, button))
+        dom.span({ className: "header-buttons" },
```

Comment:
@jasonLaster I don't think `div`/`span` have much semantic meaning in their own right, the `span` was just easier because it was one previously.